assets/js/modules/analytics-4/components/widgets/PopularProductsWidget.js

```diff
@@ -95,38 +92,49 @@ export default function PopularProductsWidget( props ) {
 			: undefined
 	);
 
-	if ( ! showWidget ) {
-		return <WidgetNull />;
-	}
-
 	const { rows = [] } = report || {};
 
-	const columns = [
-		{
-			field: 'dimensionValues',
-			Component: ( { fieldValue } ) => {
-				const [ title, url ] = fieldValue;
-				const permaLink = getFullURL( siteURL, url.value );
-
-				return (
-					<Link
-						href={ permaLink }
-						title={ title.value }
-						external
-						hideExternalIndicator
-					>
-						{ title.value }
-					</Link>
-				);
+	const columns = useInViewSelect( ( select ) => {
+		if ( ! showWidget ) {
+			return [];
+		}
+
+		return [
+			{
+				field: 'dimensionValues',
+				Component: ( { fieldValue } ) => {
+					const [ title, url ] = fieldValue;
+					const permaLink = select(
```

Comment:
This is really the `serviceURL` rather than the permalink which I'd expect to be local to the site.